lib/db.js

```diff
@@ -319,6 +340,22 @@ module.exports = function (
       )
   }
 
+  DB.prototype.keyFetchTokenVerified = function (id) {
+    log.trace({ op: 'DB.keyFetchTokenVerified', id: id })
+    return this.pool.get('/keyFetchToken/' + id.toString('hex') + '/verified')
+      .then(
+        function (result) {
+          return result;
```

Comment:
we should just switch this repo over to using semicolons while @dannycoates isn't looking :-P